download-vr-cli.yml

```diff
@@ -1,6 +1,6 @@
 steps:
   - bash: |
       set -exuo pipefail
-      npm install -g vr-approval-cli@0.0.168
+      npm install -g vr-approval-cli@0.4.3
```

Comment:
FYI: this will not be used in this way going forward https://github.com/microsoft/fluentui/pull/32048